src/radical/pilot/pilot.py

```diff
@@ -113,13 +114,17 @@ def __init__(self, pmgr, descr):
         self._session_sandbox  = self._session._get_session_sandbox (pilot)
         self._pilot_sandbox    = self._session._get_pilot_sandbox   (pilot)
         self._client_sandbox   = self._session._get_client_sandbox()
+        self._pilot_sandbox    = self._session._get_pilot_sandbox   (pilot)
+        self._pilot_fs         = self._session._get_pilot_fs   (pilot)
 
         # contexts for staging url expansion
         # NOTE: no task sandboxes defined!
         self._rem_ctx = {'pwd'     : self._pilot_sandbox,
                          'client'  : self._client_sandbox,
                          'pilot'   : self._pilot_sandbox,
-                         'resource': self._resource_sandbox}
+                         'resource': self._resource_sandbox
+                         'pilot_fs': self._pilot_fs,
```

Comment:
`pilot_fs` is the `scheme` name. Please suggest better names. 
idk if it's clarifying enough. 
@andre-merzky @mtitov 

eg. 
pilot_remote_fs
pilot_fs_endpoint
pilot_fs_abs
pilot_fs_remote

Reply:
`resource` would be the perfect name IMHO - but alas we use that already for the resource sandbox :-/  Let's leave it as is for now I guess...

Reply:
As I understand it, this scheme maps to the `<resource_domain>.<resource_name>.<access_schema>.filesystem_endpoint` field in the Session resources, correct? I think the name should try to make this connection clear. `endpoint:///` would make sense, since we don't expect the pilot to be able to resolve `endpoint://<access_schema>.<resource_name>.<resource_domain>/` for anything but itself. `filesystem_endpoint://` seems kind of long. `fs_endpoint://` seems like a potentially confusing abbreviation.

`pilot_fs://` does not do much to help me intuit that we are talking about `<resource_domain>.<resource_name>.<access_schema>.filesystem_endpoint`, so if you go with this choice, I think there is extra burden on documentation. (Please be clear and thorough.)